src/parquet/column/properties.h

```diff
@@ -78,10 +78,12 @@ class PARQUET_EXPORT ReaderProperties {
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
 static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
+static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
```

Comment:
We will have to add another member for the dictionary fallback encoding anyway.  I restored this to keep the API more intuitive with respect to enable/disable dictionary encoding. We can use the `encoding_` for value/fallback encoding. Thoughts ?